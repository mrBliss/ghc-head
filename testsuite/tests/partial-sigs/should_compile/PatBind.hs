{-# LANGUAGE PartialTypeSignatures #-}
module PatBind where

foo :: _
Just foo = Just id

test :: (Bool, Char)
test = (foo True, foo 'x')
