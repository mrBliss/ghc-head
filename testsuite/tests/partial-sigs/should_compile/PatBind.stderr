TYPE SIGNATURES
  foo :: forall a. a -> a
TYPE CONSTRUCTORS
COERCION AXIOMS
Dependent modules: []
Dependent packages: [base, ghc-prim, integer-gmp]
